ui/src/app/kvm/views/KVMDetails/KVMSummary/KVMAggregateResources/KVMAggregateResources.tsx

```diff
@@ -3,13 +3,92 @@ import React, { useEffect } from "react";
 import { useDispatch, useSelector } from "react-redux";
 
 import type { RootState } from "app/store/root/types";
+import type { Pod } from "app/store/pod/types";
+import type { KVMResourcesCardProps } from "app/kvm/components/KVMResourcesCard";
 import { actions as podActions } from "app/store/pod";
 import podSelectors from "app/store/pod/selectors";
 import KVMResourcesCard from "app/kvm/components/KVMResourcesCard";
 import { formatBytes } from "app/utils";
 
 type Props = { id: number };
 
+/**
+ * Normalise pod data for use in KVMResourcesCard component.
+ * @param pod - the pod whose resources are to be normalised
+ */
+const normaliseResources = (pod: Pod): KVMResourcesCardProps => {
```

Comment:
The name of this type feels a little bit odd here, given it has nothing to do with react in this context. Would it be clearer if both this component and KVMResourceCard imported a type called `KVMResources` which could be aliased to `KVMResourcesCardProps` in `KVMResourcesCard`?